localisation/00_zuwc_united_localisation.csv

```diff
@@ -2926,17 +2926,18 @@ RULE_SCOURGE_SPAWN_RANDOM;Random;;Zuf
+has_realm_province_with_5_or_more_settlements_tooltip;Has 5 or more holdings in your realm. ;;Verf�gt �ber 5 oder mehr Besitzungen in Eurem Reich. ;;;;;;;;;;;x
+create_gorian_empire_tt;�Y[Root.GetBestName]�! creates the �Y[e_gorian_empire.GetName]�!;;�Y[Root.GetBestName]�! gr�ndet das �Y[e_gorian_empire.GetName]�!;;;;;;;;;;;x
+EVTTITLE_WCOE_100;Reborn in Purple;;In Purpur wiederauferstanden;;;;;;;;;;;x
+EVTDESC_WCOE_100;The imperial ogres were the greatest power on Draenor, before the Fel tide of the rising Horde swept away all its glory in one decisive blow. Today, the many tribes of the ogres gathered in the great halls of [target_imperator.GetCapital] to discuss the future of their dwindling race. The elders formed a new senate and acknowledged their gracious host and protector, [target_imperator.GetFirstName], as the new [target_imperator.GetTitle] of the ogres. In [target_imperator.GetHerHis] speech to the many tribesmen gathered outside of the senate hall, [target_imperator.GetTitledFirstName] is raised high on the shields of [target_imperator.GetHerHis] warriors and [target_imperator.GetSheHe] proclaims a new destiny for [target_imperator.GetHerHis] people.\n"No longer shall we live as savages in the backwater corners of this world. WE are the children of old Draenor. WE will forge a new empire and smash all those, who oppose us!" With these words and the cheers of thousands, the purple-gold banners of old are unfurled in the sky. Thus, the Gorian Empire is reborn and the ogres finally claim their rightful place among the proud races of Azeroth.;;Die Oger waren die gr��te Macht Draenors, bevor die Flut der aufstrebenden Horde ihren Ruhm mit einem entscheidenden Schlag hinwegfegte. Heute versammelten sich die vielen St�mme der Oger in den  Hallen von [target_new_goria.GetName], um �ber die Zukunft ihrer schwindenden Rasse zu diskutieren. Die �ltesten bildeten einen neuen Senat und best�tigten ihre[target_imperator.GetnMaleOpp] gn�dige[target_imperator.GetnMaleOpp] Gastgeber[target_imperator.GetinFemale] und Besch�tzer[target_imperator.GetinFemale], [target_imperator.GetFirstName], als [target_imperator.Getdendie] neue[target_imperator.GetnMale] [target_imperator.GetTitle] der Oger. In [target_imperator.GetHerHis]er Rede verk�ndet [target_imperator.GetSheHe] ein neues Schicksal f�r [target_imperator.GetHerHis] Volk. �Wir werden nicht l�nger als Wilde in den Winkeln dieser Welt leben. WIR sind die Kinder der alten Draenor. WIR werden ein neues Imperium schmieden und all jene zerschlagen, die sich uns widersetzen!� Mit diesen Worten werden unter tosendem Jubel die purpur-goldenen Banner der alten Welt am Himmel entrollt. Das gorianische Imperium ist wiedergeboren und die Oger beanspruchen  ihren rechtm��igen Platz unter den stolzen V�lkern Azeroths.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCOE_100;The imperial ogres were the greatest power on Draenor, before attempting to subjugate the elements themselves. The Gorian Empire ignited conflict with the orcs over the subjugation and eventually the elements, hearing the pleas of the orcs, unleashed their power and shattered the ogre empire. Now today, many ogres gathered in the great hall of [target_imperator.GetCapital] to discuss the future of their beleaguered race. The elders formed a senate and acknowledged their host, [target_imperator.GetFirstName], as the [target_imperator.GetTitle] of the ogres. In [target_imperator.GetHerHis] speech to the many clansmen gathered outside of the hall,[target_imperator.GetSheHe] proclaims a new destiny for [target_imperator.GetHerHis] people.\n"No longer will we live in mounds dug in the backwater corners of this world. Subservient to the whims of others. Enslaved to their wills and stripped of who we are.  WE ruled Draenor. WE forged an empire. WE took on the world and here we stand. Smash those who oppose us! Glory to the Ogre Empire!" With these words the gathered clansmen cheered, the purple-gold banners of old unfurled in the sky, and drums rumbled to life. The Gorian Empire is reborn. Finally, will the ogres claim their rightful place among the peoples of Azeroth..;;Die Oger waren die gr��te Macht Draenors, bevor die Flut der aufstrebenden Horde ihren Ruhm mit einem entscheidenden Schlag hinwegfegte. Heute versammelten sich die vielen St�mme der Oger in den  Hallen von [target_new_goria.GetName], um �ber die Zukunft ihrer schwindenden Rasse zu diskutieren. Die �ltesten bildeten einen neuen Senat und best�tigten ihre[target_imperator.GetnMaleOpp] gn�dige[target_imperator.GetnMaleOpp] Gastgeber[target_imperator.GetinFemale] und Besch�tzer[target_imperator.GetinFemale], [target_imperator.GetFirstName], als [target_imperator.Getdendie] neue[target_imperator.GetnMale] [target_imperator.GetTitle] der Oger. In [target_imperator.GetHerHis]er Rede verk�ndet [target_imperator.GetSheHe] ein neues Schicksal f�r [target_imperator.GetHerHis] Volk. �Wir werden nicht l�nger als Wilde in den Winkeln dieser Welt leben. WIR sind die Kinder der alten Draenor. WIR werden ein neues Imperium schmieden und all jene zerschlagen, die sich uns widersetzen!� Mit diesen Worten werden unter tosendem Jubel die purpur-goldenen Banner der alten Welt am Himmel entrollt. Das gorianische Imperium ist wiedergeboren und die Oger beanspruchen  ihren rechtm��igen Platz unter den stolzen V�lkern Azeroths.;;;;;;;;;;;x
```

Reply:
![grafik](https://user-images.githubusercontent.com/47716948/82999008-ba0fc680-a008-11ea-8ebd-8f1dc95185b4.png)
But it was the fel horde, that destroyed the empire. The first war against the orcs and the elements was hundreds of years prior to that...

Reply:
would be better to put the last sentence like this "Finally, the ogres will claim their rightful place among the peoples of Azeroth." But personally, I like the previous version more...
Would ogres really call their empire the "ogre empire"? I don't think so...

Reply:
I agree with you on the readjustment Arithon for the sentence! I do believe they would call it an Ogre Empire. There is a precedent for it in the lore and it stresses that they are in control of their own destiny.

In regards to the horde killing Ogres. It's the Old Horde your referencing as the Fel Horde doesnt come about until many years later and is based solely in Outland.
To the whole deal of destroyed vs shattered. Shattered doesnt mean their empire was destroyed. It means it became fractured, which is the case. Further up on that page it references them unable to maintain their Empire during the siege. Effectively speaking at that time the Empire is shattered. They rebuild the structure centered around Highmaul but the Gorian Empire isnt in its glory days anymore. It's been shattered to a less immense state. The Shattering is what the book intro to Cataclysm is called. Azeroth was in fact shattered, not destroyed.

Reply:
Speaking for myself, I agree with Arithon on the ogre empire. I feel like thats something other races would call them, whereas they themselves would stick to calling it the Gorian Empire. It would be like calling the Empire of Arathor the human empire, or the Zandalari Empire the troll empire. 

On the shattered/ destroy, I like shatter more. Fragments of the empire survived, and we know of several ogres who were attempting to unite them post collapse, presumably in memory of the empire (look at ogri'la or High King Maulgar).